src/components/CurrentUserPersonalDetailsSkeletonView/index.tsx

```diff
@@ -23,34 +22,32 @@ function CurrentUserPersonalDetailsSkeletonView({shouldAnimate = true, avatarSiz
     const StyleUtils = useStyleUtils();
     const avatarPlaceholderSize = StyleUtils.getAvatarSize(avatarSize);
     const avatarPlaceholderRadius = avatarPlaceholderSize / 2;
-    const spaceBetweenAvatarAndHeadline = styles.mb3.marginBottom + styles.mt1.marginTop + (variables.lineHeightXXLarge - variables.fontSizeXLarge) / 2;
-    const headlineSize = variables.fontSizeXLarge;
-    const spaceBetweenHeadlineAndLabel = styles.mt1.marginTop + (variables.lineHeightXXLarge - variables.fontSizeXLarge) / 2;
-    const labelSize = variables.fontSizeLabel;
+    const x = 30;
+
     return (
         <View style={styles.avatarSectionWrapperSkeleton}>
             <SkeletonViewContentLoader
                 animate={shouldAnimate}
                 backgroundColor={theme.skeletonLHNIn}
                 foregroundColor={theme.skeletonLHNOut}
-                height={avatarPlaceholderSize + spaceBetweenAvatarAndHeadline + headlineSize + spaceBetweenHeadlineAndLabel + labelSize}
+                height={avatarPlaceholderSize + styles.pb5.paddingBottom}
```

Comment:
Question for this whole set of changes - are we changing this skeleton view everywhere? do we need to maintain the old style (big circle and lines underneath) as well?

Reply:
we don't need the old style. 
this skeleton view is used in only one place

Reply:
i renamed the file to `AccountSwitcherSkeletonView`.